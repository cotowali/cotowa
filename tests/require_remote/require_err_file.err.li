require 'https://raw.githubusercontent.com/cotowali/cotowali/main/tests/require_remote/err.mod.li'
require 'github:cotowali/cotowali/main/tests/require_remote/err.mod.li'
