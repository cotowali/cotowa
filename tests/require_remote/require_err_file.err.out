error: err.mod.li 1,5: mismatched types: `int` (expected) and `string` (found)
    1 | var n: int = '10'
      |     ^^^^^^^^^^^^^
error: err.mod.li 1,5: variable `n` is already defined
    1 | var n: int = '10'
      |     ^
error: err.mod.li 1,5: mismatched types: `int` (expected) and `string` (found)
    1 | var n: int = '10'
      |     ^^^^^^^^^^^^^
